app/src/main/java/com/example/tasky/agenda_details/presentation/AgendaDetailsViewModel.kt

```diff
@@ -334,19 +339,165 @@ class AgendaDetailsViewModel @Inject constructor(
         }
     }
 
-    fun save() {
+    private fun updateEvent() {
+        val eventId = agendaItemId ?: throw IllegalArgumentException("agendaItemId is null")
+        val fromInEpochSeconds =
+            DateTimeHelper.getEpochMillisecondsFromDateAndTime(
+                date = _viewState.value.fromDate,
+                time = _viewState.value.fromTime
+            )
+        val toInEpochSeconds =
+            DateTimeHelper.getEpochMillisecondsFromDateAndTime(
```

Comment:
What now, epoch seconds or milliseconds? The variable says something different than the function

Reply:
will fix ! needs to be ms